### DISCUSSION TITLE
Hide Linktools

### DISCUSSION THREAD

**Discussion by CodeSippr** [clientIO/joint#2576] [Q&A]
### Introduction

Hello, I'm using the built-in wires to connect logical gates. When hovering over these wires, the link tools (delete and change source/target) are displayed. Is there a simple way to prevent this hover event?

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS

**Comment by kumilingus** [comment]
Are you using `v3`? 
You should switch to `standard.Link` and add the [tools](https://resources.jointjs.com/tutorial/link-tools) only when needed.
You can also prevent the legacy link tools from appearing via CSS (use `display: none` for instance).

Note that the links with built-in tools and hover effect has been removed from `v4`.
